src/main.cpp

```diff
@@ -8,10 +8,77 @@
 #include <iostream>
 #include "window.hpp"
 
+#include "../include/model.hpp"
 #include "raylib.h"
 #include "shape3d.hpp"
 #include "Camera.hpp"
 
+void test_raylib_model(void)
```

Comment:
All cpp functions should be in Lower Camel Case. Also, better put test functions in the test_functions directory. Its on "dev" branch, think about  merging dev on your own branch before doing the PR if not already done